commands/remove.rb

```diff
@@ -16,7 +16,7 @@ def self.remove(pkg, verbose = CREW_VERBOSE)
     end
 
     # Do not remove any packages in CREW_ESSENTIAL_PACKAGES, as those are needed for ruby and thus crew to run.
-    if CREW_ESSENTIAL_PACKAGES.include?(pkg.name) && !CREW_FORCE
+    if CREW_ESSENTIAL_PACKAGES.include?(pkg.name) && !force
```

Comment:
`&& (!force || !CREW_FORCE)`?  Otherwise, the environment variable serves no purpose.

Reply:
CREW_FORCE isn't set by an environment variable-- it's set by checking for the `-f` argument, and right now its only use is in `lib/downloader.rb`.

Reply:
i.e. these changes do not affect user-facing behavior at all.

Reply:
I should have said constant instead of environment variable, actually.  Wouldn't it make sense to pass parameters like below?
```
def self.remove(pkg, verbose = CREW_VERBOSE, force = CREW_FORCE)
```

Reply:
You could still do that with:
```
 def self.remove(pkg, verbose: CREW_VERBOSE, force: CREW_FORCE)
```

Reply:
True, but I'd rather we pass the information through parameters than have a default value set by an unpredictable constant in an external file. Ideally, once we've split every command out into a file, we won't have any need for these constants, because we'll pass the parameters through in calls to every command.

Reply:
Any thoughts on this @satmandu?